scripts/start_onboard.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+cd owr_software/rover/
+start/tmuxprocmon
```

Comment:
running `owr_software/rover/start/tmuxprocmon` would do exactly the same thing as this why do you need a script?

Reply:
Yeah I would make it a symbolic link at some point. The script is there for reference so that setting up a new computer to be the groundstation is easier and can just be copied.
The idea was to track this file in git.